src/Subjects/MediaSubject.php

```diff
@@ -85,18 +86,18 @@ public function setUp($serial)
         $this->skuEntityIdMapping = isset($status[RegistryKeys::SKU_ENTITY_ID_MAPPING]) ? $status[RegistryKeys::SKU_ENTITY_ID_MAPPING] : array();
 
         // initialize media directory => can be absolute or relative
-        if ($this->getConfiguration()->hasParam(ConfigurationKeys::MEDIA_DIRECTORY)) {
+        if ($this->getConfiguration()->hasParam(FileUploadConfigurationKeys::MEDIA_DIRECTORY)) {
             try {
-                $this->setMediaDir($this->resolvePath($this->getConfiguration()->getParam(ConfigurationKeys::MEDIA_DIRECTORY)));
+                $this->setMediaDir($this->resolvePath($this->getConfiguration()->getParam(FileUploadConfigurationKeys::MEDIA_DIRECTORY)));
             } catch (\InvalidArgumentException $iae) {
                 $this->getSystemLogger()->warning($iae->getMessage());
             }
         }
 
         // initialize images directory => can be absolute or relative
-        if ($this->getConfiguration()->hasParam(ConfigurationKeys::IMAGES_FILE_DIRECTORY)) {
+        if ($this->getConfiguration()->hasParam(FileUploadConfigurationKeys::IMAGES_FILE_DIRECTORY)) {
             try {
-                $this->setImagesFileDir($this->resolvePath($this->getConfiguration()->getParam(ConfigurationKeys::IMAGES_FILE_DIRECTORY)));
+                $this->setImagesFileDir($this->resolvePath($this->getConfiguration()->getParam(FileUploadConfigurationKeys::IMAGES_FILE_DIRECTORY)));
             } catch (\InvalidArgumentException $iae) {
                 $this->getSystemLogger()->warning($iae->getMessage());
```

Comment:
Shouldn't we check here, if images has to be copied (see https://github.com/techdivision/import-category/pull/77/files#diff-6be71b8395451263d93c834bf1f5f07297af6c7b7e24e2a61447f6c761918183R146)